common/src/main/java/com/wynntils/wc/utils/ActionBarManager.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wc.utils;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.mc.event.PacketEvent.PacketReceivedEvent;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.network.chat.ChatType;
+import net.minecraft.network.protocol.game.ClientboundChatPacket;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class ActionBarManager {
+
+    private static final Pattern ACTIONBAR_PATTERN =
+            Pattern.compile("§c❤ ([0-9]+)/([0-9]+)§0\\s+(.+)\\s+§b✺ ([0-9]+)/([0-9]+)");
+
+    private static String previousActionBar = null;
+
+    private static int currentHealth = 100;
+    private static int maxHealth = 100;
+    private static int currentMana = 20;
+    private static int maxMana = 20;
```

Comment:
We assume these default values? Can we just make it null, or what is the use?

Reply:
It was arbitrary, I could do -1 like in legacy, but I guess null wouldn't be that bad either. My only concert about null is querying the health stats before the first actionbar event, like maybe the HP-bar overlay tries to render before the event. I could also use the Integer boxed type and then you could handle null from the overlay. For now i'll go with -1, if that's ok.

Reply:
Sure, -1 should be okay, though I hope it won't cause weirdness. It just has to be handled correctly, just like null.